WcaOnRails/spec/models/user_spec.rb

```diff
@@ -108,16 +94,6 @@
     expect(delegate.can_admin_results?).to be false
   end
 
-  it "allows senior delegate if board member" do
-    board_member = FactoryBot.create :delegate, :board_member
-
-    senior_delegate = FactoryBot.create :senior_delegate
-
-    expect(board_member).to be_valid
-    board_member.region_id = senior_delegate.region_id
-    expect(board_member).to be_valid
-  end
```

Comment:
The test is no longer needed because board & senior delegate are independent roles and don't have any connection once migrated to roles.